virttest/utils_windows/drive.py

```diff
@@ -22,8 +23,9 @@ def get_hard_drive_letter(session, label):
     """
     cond = "VolumeName like '%s'" % label
     try:
-        return _logical_disks(session, cond=cond, props=["DeviceID"])[0]
-    except IndexError:
+        return utils_misc.wait_for(lambda: _logical_disks(session, cond=cond,
+                                   props=["DeviceID"]), 30, 0, 1)[0]
```

Comment:
Suggest making this work for `_logical_disks`, so that all the callers can get the benefit for that.

Reply:
xuhan, thanks, has updated~